care/facility/migrations/0327_bed_is_occupied.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.11 on 2022-11-07 13:38
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("facility", "0326_auto_20221018_1526"),
+    ]
+
+    def update_previous_beds(apps, schema_editor):
+        Bed = apps.get_model("facility", "Bed")
+        ConsultationBed = apps.get_model("facility", "ConsultationBed")
+        for bed in Bed.objects.all():
+            bed.is_occupied = ConsultationBed.objects.filter(
+                bed=bed, end_date__isnull=True
+            ).exists()
+            bed.save()
```

Comment:
We could optimize the custom migration like this right?

```suggestion
        Bed.objects.all().update(
          is_occupied=F("consultationbed__end_date__isnull")
        )
```

Reply:
or even:

```suggestion
        Bed.objects.filter(consultationbed__end_date__isnull=True).update(is_occupied=True)
```